package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "data-mocks",
-  "version": "4.1.0",
+  "name": "@ovotech/data-mocks",
+  "version": "4.2.1",
   "main": "dist/mocks.js",
   "types": "dist/types.d.ts",
-  "repository": "git@github.com:grug/data-mocks.git",
+  "repository": "git@github.com:aidenscott2016/data-mocks.git",
   "author": "Dave Cooper <dave@davecooper.dev>",
   "license": "MIT",
-  "homepage": "https://github.com/grug/data-mocks",
+  "homepage": "https://github.com/aidenscott2016/data-mocks",
```

Comment:
will undo this if work will be merged upstream

Reply:
Haha why did you do this? 😂

Reply:
Bold move 😂

Reply:
ajaja I wanted to use this work immediately so I pushed it to ovo npm and changed a few things to make it obvious it's a fork